compiler-lib/compiler/commands/test.rkt

```diff
@@ -666,7 +666,13 @@
   ;; This should be in Racket somewhere and return all the collection
   ;; paths, rather than just the first as collection-path does.
   (define (collection-paths c)
-    (match-define (list-rest sc more) (map path->string (explode-path c)))
+    (define (path->string* maybe-path)
```

Comment:
I haven't seen one. The only place I've seen `'up` and `'same` used are `explode-path`, `split-path`, `build-path/convention-type`, and `build-path`. I suppose this function is the same as `(path->string (build-path maybe-path))`, But it think its clearer what this is doing. Thoughts?